src/components/work/Footer/Footer.tsx

```diff
@@ -18,16 +18,16 @@ import { BeenestSVGPrimary } from 'shared/svgComponents/SvgComponents';
 
 const contentData = [
```

Comment:
shouldn't this be in the footer PR?

Reply:
dont know how this got in here, footer PR has the updates though